### DISCUSSION TITLE
Va-icon fill

### DISCUSSION THREAD

**Discussion by OlegFromOmsk** [epicmaxco/vuestic-ui#1882] [Q&A]
Hello
How can I use `fill `option from **material icons** for va-icon?

**Comment by m0ksem** [comment]
Hi. You can extend [Icons Config](https://vuestic.dev/en/services/icons-config#font-name-pattern).

```ts
createApp(App)
  .use(VuesticPlugin, {
    icons: createIconsConfig({ 
      fonts: [
        {
          name: '{icon}-fill',
          resolve: ({ icon } => ({
            content: 'icon',
            class: 'material-symbols-outlined material-symbols-filled'
          }))
        }
      ] 
     })
  })
  .mount('#app')
```

```vue
<style>
.material-symbols-filled {
  font-variation-settings:
  'FILL' 1,
  'wght' 400,
  'GRAD' 0,
  'opsz' 48
}
</style>
```

Then any icon with -fill suffix will apply `material-symbols-filled` class.

```vue
<va-icon name="clock-fill" />
```

**Comment by OlegFromOmsk** [comment]
I have tried do this, but it doesn't work. Icon is filled

`<i aria-hidden="true" notranslate="" class="va-icon material-icons material-symbols-outlined material-symbols-filled" style="font-size: 24px;">description</i>`

```
<style>
.material-symbols-filled {
  font-variation-settings:
  'FILL' 0,
  'wght' 400,
  'GRAD' 0,
  'opsz' 48
}
</style>
```
